File: reaction.py
from pyautogui import *
import pyautogui
import time
import keyboard
import random
import win32api, win32con
from numba import njit

@njit
def click(x,y):
    win32api.SetCursorPos((x,y))
    win32api.mouse_event(win32con.MOUSEEVENTF_LEFTDOWN,0,0)
    time.sleep(0.01)
    win32api.mouse_event(win32con.MOUSEEVENTF_LEFTUP,0,0)
click(0,322)
while keyboard.is_pressed('q') == False:
    if pyautogui.pixel(0,322)[1] == 205:
        click(0,322)

File: reaction.cpp
#include <Windows.h>
#include <iostream>
#include <ctime>
#include <conio.h>

void click(int x, int y) {
    SetCursorPos(x, y);
    mouse_event(MOUSEEVENTF_LEFTDOWN, 0, 0, 0, 0);	
    mouse_event(MOUSEEVENTF_LEFTUP, 0, 0, 0, 0);
}

int main() {
    while (!GetAsyncKeyState('Q')) {
        if (GetPixel(GetDC(NULL), 0, 322) == RGB(113, 205, 113)) {
            click(0, 322);
        }
    }
    return 0;
}
